apps/graphql/src/apps/common/types/outputs/RouteStop.js

```diff
@@ -28,7 +28,7 @@ export default new GraphQLObjectType({
         if (code == null) {
           return null;
         }
-        const stops = await dataLoader.locations.load({ code });
+        const stops = await dataLoader.locations.byID.load({ code });
         if (!Array.isArray(stops) || stops.length < 1) {
           return null;
         }
```

Comment:
I see that the return type of the dataloader is changed, which is good, I see that this doesn't to give any flow-error which is bad. I want to get rid of this as well. 

```
 if (!Array.isArray(stops) || stops.length < 1) {
          return null;
  }
```